app/tests/Fixture/ArticlesFixture.php

```diff
@@ -0,0 +1,34 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Test\Fixture;
+
+use Cake\TestSuite\Fixture\TestFixture;
+
+/**
+ * ArticlesFixture
+ */
+class ArticlesFixture extends TestFixture
+{
+    /**
+     * Init method
+     *
+     * @return void
+     */
+    public function init(): void
+    {
+        $this->records = [
+            [
+                'id' => 1,
+                'user_id' => 1,
+                'title' => 'Lorem ipsum dolor sit amet',
+                'slug' => 'Lorem ipsum dolor sit amet',
+                'body' => 'Lorem ipsum dolor sit amet, aliquet feugiat. Convallis morbi fringilla gravida, phasellus feugiat dapibus velit nunc, pulvinar eget sollicitudin venenatis cum nullam, vivamus ut a sed, mollitia lectus. Nulla vestibulum massa neque ut et, id hendrerit sit, feugiat in taciti enim proin nibh, tempor dignissim, rhoncus duis vestibulum nunc mattis convallis.',
+                'published' => 1,
+                'created' => '2023-11-01 04:41:51',
+                'modified' => '2023-11-01 04:41:51',
+            ],
+        ];
```

Comment:
As I mentioned https://github.com/sergio9977/tutorial-cakephp/pull/2#discussion_r1378746883, we plan not to focus on the UT. 

The fixture file generated by the bake command generates the records randomly. It's become hard to test; therefore, we must prepare proper data for test.

Reply:
Similar comment https://github.com/sergio9977/tutorial-cakephp/pull/2#discussion_r1382339260, it's OK to leave the generated fixtures and written unit test for now.